packages/sku/src/services/vite/createConfig.ts

```diff
@@ -0,0 +1,68 @@
+import { createRequire, builtinModules } from 'node:module';
+import type { InlineConfig } from 'vite';
+
+import react from '@vitejs/plugin-react';
+import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin';
+
+import type { SkuContext } from '@/context/createSkuContext.js';
+
+const require = createRequire(import.meta.url);
+
+const clientEntry = require.resolve('./entries/vite-client.jsx');
+
+export const createViteConfig = ({
+  skuContext,
+  configType = 'client',
+  plugins = [],
+}: {
+  skuContext: SkuContext;
+  configType?: 'client';
+  plugins?: InlineConfig['plugins'];
+}) => {
+  const outDir = {
+    client: 'dist',
+  };
+
+  return {
+    root: process.cwd(),
+    plugins: [
+      react(),
+      vanillaExtractPlugin({
+        unstable_pluginFilter: (plugin) =>
+          !plugin.name.startsWith('vite:react-'),
+      }),
```

Comment:
As per my comment on your previous PR #1178, this shouldn't be necessary as the VE plugin no longer passes through any plugins by default.